config/crontab-example

```diff
@@ -46,8 +46,8 @@ MAILTO=!!(*= $mailto *)!!
 # Once a week (early Monday morning)
 37 8 * * 1 !!(*= $user *)!! !!(*= $vhost_dir *)!!/!!(*= $vcspath *)!!/commonlib/bin/run-with-lockfile.sh -n !!(*= $vhost_dir *)!!/send-pro-metrics-report.lock !!(*= $vhost_dir *)!!/!!(*= $vcspath *)!!/script/send-pro-metrics-report || echo "stalled?"
 
-# Once a month
-42 1 * 1-12 * !!(*= $user *)!! !!(*= $vhost_dir *)!!/!!(*= $vcspath *)!!/script/download-geoip-data
+# Once a week (on a Wednesday evening)
+42 23 * * 4 !!(*= $user *)!! !!(*= $vhost_dir *)!!/!!(*= $vcspath *)!!/script/download-geoip-data
```

Comment:
https://crontab.guru/#42_23_*_*_4 suggests that this will run on _Thursday_?

Reply:
Sigh, yes - my mistake. Brain clearly not functioning properly.